src/@sdk/index.ts

```diff
@@ -158,33 +202,38 @@ export class SaleorAPI {
         }
       );
 
-      const subscription = observable.subscribe(
-        result => {
-          const { data, errors: apolloErrors } = result;
-          const errorHandledData = handleDataErrors(
-            mapFn,
-            data as any,
-            apolloErrors
-          );
-          if (onUpdate) {
-            if (errorHandledData.errors) {
-              if (onError) {
-                onError(errorHandledData.errors);
-              }
-            } else {
-              onUpdate(errorHandledData.data as TResult);
-              if (onComplete) {
-                onComplete();
+      let unsubscribe = null;
+
+      if (!options.skip) {
+        const subscription = observable.subscribe(
+          result => {
+            const { data, errors: apolloErrors } = result;
+            const errorHandledData = handleDataErrors(
+              mapFn,
+              data as any,
+              apolloErrors
+            );
+            if (onUpdate) {
```

Comment:
This if statement can be simplified to if(onUpdate && (errorHandledData.errors || onError) (if onError maybe be `true` only if errorHandledData.errors is also true)

Reply:
I'm not convinced that we should change this. First of all, we want to check the output data and then based on the provided props run a particular piece of code. In my opinion, these conditions are enough readable.